tests/Builder/Expression/Pipelines.php

```diff
@@ -10,4 +10,115 @@
 
 enum Pipelines: string
 {
+    /** Use in $addFields Stage */
+    case FirstUseInAddFieldsStage = <<<'JSON'
+    [
+        {
+            "$addFields": {
+                "firstItem": {
+                    "$first": "$items"
+                }
+            }
+        }
+    ]
+    JSON;
+
+    /**
+     * Example
+     *
+     * @see https://www.mongodb.com/docs/manual/reference/operator/aggregation/firstN-array-element/#example
+     */
+    case FirstNExample = <<<'JSON'
+    [
+        {
+            "$addFields": {
+                "firstScores": {
+                    "$firstN": {
+                        "n": {
+                            "$numberInt": "3"
+                        },
+                        "input": "$score"
+                    }
+                }
+            }
+        }
+    ]
+    JSON;
+
+    /** Use in $addFields Stage */
```

Comment:
Where are these comments coming from, and why do some of the cases in this enum have entirely different doc block comments? For instance, the singular "Example" descriptions look redundant.

Reply:
I just stupidly copy-pasted the examples from the docs. We can improve each 
description as needed.

The one you are pointing isn't from the doc, that's why there is not link.